core/src/main/java/com/graphhopper/routing/util/BikeCommonFlagEncoder.java

```diff
@@ -434,12 +444,17 @@ int getSpeed( OSMWay way )
             } else 
             {
                 if (way.hasTag("bicycle", "designated") || way.hasTag("bicycle", "official"))
-                    speed = getHighwaySpeed("cycleway");
-                else                
-                    speed = PUSHING_SECTION_SPEED * 2;
-            }
-        }
-
+                {
```

Comment:
In these cases here we (also) have just pushing sections ... is the complex if handling really necessary or can you simplify this?